app/views/supervisor_reports/_show.haml

```diff
@@ -102,14 +110,18 @@
     %p
       %strong= human_name :supervisor_report, :employee_arrived_to_test_at
       = report.employee_arrived_at_test_at.strftime '%l:%M %P'
-  - if report.test_started_at.present?
```

Comment:
Interesting. This was requested?

Do we need to change the PDF as well?

Reply:
This was unintentional on my part.